### PR TITLE
feat: Create chart for lily based on latest visor

### PR DESCRIPTION
**Pull request by placer14**
This PR:
- creates chart Lily v0.1
- updates instances of visor -> lily

Note: Default values have changed. Review values using this chart before deployment.

### CODE REVIEW

File: charts/lily/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        [Storage.Postgresql.{{ .name }}]
+          SchemaName = {{ .schema | default "lily" | quote }}
+          URLEnv = "LILY_STORAGE_POSTGRESQL_{{ .name | upper }}_URL"
+          ApplicationName = {{ .applicationName | default (include "sentinel-lily.instanceName" $ ) | quote }}
+          PoolSize = {{ .poolSize | default 20 }}
+          AllowUpsert = {{ .allowUpsert | default false }}
+        {{- end }}
+      {{- end }}
+      {{- if .Values.daemon.storage.file }}
+      [Storage.File]
```
Comment (iand) [inline charts/lily/templates/configmap.yaml:35]: We don't need the file storage here

**Review by iand — APPROVED** [review]
Quite difficult to review. I'm approving on the basis that this is derived from the already-working visor chart

**Comment by placer14** [comment]
@iand I should've mentioned it, but I separated the replication and modification steps as independent commits. If this is inadequate, what do you recommend for next time to simply the review? (thanks for taking a look)

File: charts/lily/templates/configmap.yaml (hunk 1 above)
Comment (placer14) [reply charts/lily/templates/configmap.yaml:35]: As a generic configuration script, wouldn't it be appropriate to offer all of the features? Or are you suggesting we don't need it for another reason?

**Pull request merged**